src/TraceEvent/Computers/SampleProfilerThreadTimeComputer.cs

```diff
@@ -41,6 +41,13 @@ public SampleProfilerThreadTimeComputer(TraceLog eventLog, SymbolReader symbolRe
         /// If set we compute thread time using Tasks
         /// </summary>
         public bool UseTasks;
+
+        /// <summary>
+        /// Track additional info on like EventName or so.
+        /// Default to true to keep backward compatibility.
+        /// </summary>
+        public bool TrackAdditionalInfo = true;
```

Comment:
Lets call it IncludeEventSourceEvents -  TrackAdditionalInfo is too generic, the name I am suggesting gives people more information about what to expect.

Reply:
Sure. Fix in the next iteration.